backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/ContextUtils.kt

```diff
@@ -146,7 +146,7 @@ internal interface ContextUtils : RuntimeAware {
     fun isExternal(descriptor: DeclarationDescriptor): Boolean {
         val pkg = descriptor.findPackage()
         return when (pkg) {
-            is IrFile -> false
+            is IrFile -> if (pkg.packageFragmentDescriptor.containingDeclaration != context.moduleDescriptor) true else false
```

Comment:
`if (condition) true else false` can be simplified to just `condition`.

Reply:
done